src/Meta/SchemaManager.php

```diff
@@ -44,23 +47,10 @@ class SchemaManager implements IteratorAggregate
     public function __construct(ConnectionInterface $connection)
     {
         $this->connection = $connection;
-        $this->boot();
-    }
 
-    /**
-     * Load all schemas from this connection.
-     */
-    public function boot()
-    {
         if (! $this->hasMapping()) {
             throw new RuntimeException("There is no Schema Mapper registered for [{$this->type()}] connection.");
         }
-
-        $schemas = forward_static_call([$this->getMapper(), 'schemas'], $this->connection);
```

Comment:
Why did you remove these lines?

Reply:
Please add them back. It's essential for the mapping process.

Reply:
Except they render the code useless. When they are in, the code uses all schemas and generates all the tables from all the schemas, at least when using PostgreSQL. I tested it with and without on both and the functionality worked on both MySQL and PostgreSQL with the changes I made.
I'll do another round of testing, but I'm fairly certain I'll get the same result.

Reply:
It's important to note that I am running the latest Laravel Nova, and the behavior I experienced is using that code base.

Reply:
You wouldn't notice the difference unless PostgreSQL supports multiple database in a single database deamon as MySQL does. For instance, you can have a MySQL database `a` with a table `a_table` and a database `b` with a table `b_table` having a foreign key pointing to `a.a_table`. 😅 It's not easy to make a good example.